testing/e2e/workload_test.go

```diff
@@ -516,6 +517,145 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			Name:         "Do not update the created workload with `merge` update strategy",
```

Comment:
name of the test in confusing. `create workload with merge strategy`

Reply:
this workload is already created, and I created a file to see if it would delete some of the fields

Reply:
Then use Update instead of create

Reply:
renamed